### PR TITLE
📚: add note convention mod

### PR DESCRIPTION
**Pull request by EstEarth202**

### CODE REVIEW

File: src/certified_datapacks/requirement.md [hunk 1]
```diff
@@ -13,3 +13,12 @@ A datapack must satisfy these requirements to receive the Certified Datapack tit
...
+
+### Note from Convention Mod
+
+- **Learn more about the convention** by reading the `Note from Convention Mod (Optional)` section, this allows you to confidently inspect your datapack before requesting review and reduces the Datapack Reviewer's workload.
+- **Request review:** please include every website where your datapack is published. It doesn't have to be just `GitHub` or `Planetminecraft`. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
+- **Datapack unreleased:** please don't DM the file; instead, use `GitHub private repo` and invite the reviewer. Don't worry about any datapack functions leaking.
+- **After certified:** then your datapack will have an updated version, which you can request to review as many times as you want.
+- **Request review again:** please add the `changelog` or the `location of the changed file`. And If you're using `GitHub`, we can easily check the `commit history`. By doing so, the reviewer can't waste time reviewing what they've already reviewed.
+- If you have any questions about convantion, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```
Comment (EstEarth202) [inline src/certified_datapacks/requirement.md]: 🚨: fix typo "convantion"

Comment (EstEarth202) [reply src/certified_datapacks/requirement.md]:
```suggestion
- If you have any questions about convention, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```

File: src/certified_datapacks/requirement.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+# Requirement
+
+A datapack must satisfy these requirements to receive the Certified Datapack title
+
+- Datapack must be portable, it must not be tied down to a specific save.
+- Datapack must follow all of **Tier 3** Conventions.
+- Use `core`, `library`, `API`, `utils`, etc. These datapacks must also be certified.
+  - Your review status will be paused until those datapacks are also certified.
+- If you are not the owner, **don't `steal`, `copy`, or `claim`** that you created it.
+  - `Modification` or `further develop/expand/build on` should first check the **license** or **contact the owner**, and remember to **credit**.
```
Comment (oOBoomberOo) [inline src/certified_datapacks/requirement.md:10]: Poorly worded and excessive use of code block.

File: src/certified_datapacks/requirement.md [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
+- **Request review:** please include every website where your datapack is published. It doesn't have to be just `GitHub` or `Planetminecraft`. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
+- **Review type:** please also specify the type of review you require.
+  -  **Certified Compatibility ✅ Only**
+  -  **Certified Compatibility ✅ + Style Guideline Followed 📚 (Optional)**
+- **Datapack unreleased:** please don't DM the file; instead, use `GitHub private repo` and invite the reviewer. Don't worry about any datapack functions leaking.
+- **After certified:** then your datapack will have an updated version, which you can request to review as many times as you want.
+- **Request review again:** please add the `changelog` or the `location of the changed file`. And If you're using `GitHub`, we can easily check the `commit history`. By doing so, the reviewer can't waste time reviewing what they've already reviewed.
+- If you have any questions about convention, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```
Comment (oOBoomberOo) [inline src/certified_datapacks/requirement.md:27]: Again, excessive use of bullet points and code blocks. Try to make the sentence flow naturally in paragraphs.

File: src/conventions/datapack_advancement.md [hunk 4]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
...
 
 > Keep in mind that if you are planning to release more datapacks under this name, you should use the normal syntax over this one.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
+
+**3. Datapack Advancement**
+
+Choose the normal syntax when you have more than one datapack under the namespace.
```
Comment (oOBoomberOo) [inline src/conventions/datapack_advancement.md]:
```suggestion
Use the regular format when you have more than one datapack under the namespace.

```

File: src/conventions/datapack_advancement.md [hunk 5]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
...
+
+> the advancement *should* be inside `/data/<namespace>/advancements/<datapack name>.json`.
+
+Example Datapack: [Boomber360](https://www.planetminecraft.com/member/boomber360/).
+
+**Extra: Standalone Datapack**
+
+Use standalone format when you are going only to have one datapack under the namespace.
```
Comment (oOBoomberOo) [inline src/conventions/datapack_advancement.md]:
```suggestion
Use the standalone format if you only want one datapack under the namespace.

```

File: src/conventions/datapack_advancement.md [hunk 6]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
 To implement this you simply skip the [#2: Namespace Advancement](#2-namespace-advancement) step and make sure that in step [#3: Datapack Advancement](#3-datapack-advancement) you create your file inside the `/data/global/advancements/standalone/` directory.
 
 > Keep in mind that if you are planning to release more datapacks under this name, you should use the normal syntax over this one.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
+
+**3. Datapack Advancement**
```
Comment (oOBoomberOo) [inline src/conventions/datapack_advancement.md]: Title on lines 124 and 132 appear to be unrelated

File: src/certified_datapacks/requirement.md [hunk 7]
```diff
@@ -1,11 +1,27 @@
...
+  - Your review status will be paused until those datapacks are also certified.
+- If you are not the owner, **don't `steal`, `copy`, or `claim`** that you created it.
+  - `Modification` or `further develop/expand/build on` should first check the **license** or **contact the owner**, and remember to **credit**.
+
+## Note
+
+1. We may rule a datapack out at our discretion, for reasons we would then specify.
+2. This requirement is not final and can be changed in the future.
+
+### Note from Convention Mod
```
Comment (oOBoomberOo) [inline src/certified_datapacks/requirement.md:17]: Consider a different title

File: src/conventions/datapack_advancement.md [hunk 8]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
 To implement this you simply skip the [#2: Namespace Advancement](#2-namespace-advancement) step and make sure that in step [#3: Datapack Advancement](#3-datapack-advancement) you create your file inside the `/data/global/advancements/standalone/` directory.
 
 > Keep in mind that if you are planning to release more datapacks under this name, you should use the normal syntax over this one.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
```
Comment (oOBoomberOo) [inline src/conventions/datapack_advancement.md:122]: Consider a different title

File: src/style_guideline/reloading_message.md [hunk 9]
```diff
@@ -2,4 +2,31 @@
 
 You **must not** send an automatic message when the world is reloaded. It will clutter the chat unnecessarily and create a bad experience for the users.
 
-Most of the time, the message will be an installation message, in which case you can use [Datapack Advancement](../conventions/datapack_advancement.md) instead.
\ No newline at end of file
+Most of the time, the message will be an installation message, in which case you can use [Datapack Advancement](../conventions/datapack_advancement.md) instead.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
```
Comment (oOBoomberOo) [inline src/style_guideline/reloading_message.md:9]: Consider a different title

File: src/style_guideline/reloading_message.md [hunk 10]
```diff
@@ -2,4 +2,31 @@
...
+  }
+}
+```
+> **Case 2:** debugging reload messages that normal players shouldn't see.
+
+``convention:datapack/setup``
+```mcfunction
+tellraw @a[name=Owner] <...>
+tellraw @a[tag=convention.debug] <...>
+```
```
Comment (oOBoomberOo) [inline src/style_guideline/reloading_message.md]: Lack of text description; fit this inside a paragraph instead of putting it under a title header. Don't rely on code examples alone.

**Review by oOBoomberOo — CHANGES_REQUESTED** [review]

File: src/certified_datapacks/requirement.md (hunk 2 above)
Comment (EstEarth202) [reply src/certified_datapacks/requirement.md:10]:
```suggestion
- Use `core`, `library`, `API`, `utils`, etc. These datapacks must also be certified.
  - Your review status will be paused until those datapacks are also certified.
- You must be the owner or team leader of the datapack project. 
  - "Modification" or "further develop/expand/build on" should check **the license** or **contact the owner** first. And **don't forget to give credit**.
```

File: src/conventions/datapack_advancement.md (hunk 8 above)
Comment (EstEarth202) [reply src/conventions/datapack_advancement.md:122]:
```suggestion
### **Additional Guideline (Optional)**

```

File: src/style_guideline/reloading_message.md (hunk 9 above)
Comment (EstEarth202) [reply src/style_guideline/reloading_message.md:9]:
```suggestion
### **Additional Guideline (Optional)**
```

File: src/certified_datapacks/requirement.md (hunk 7 above)
Comment (EstEarth202) [reply src/certified_datapacks/requirement.md:17]:
```suggestion
### Additional Requirement
```

File: src/conventions/datapack_advancement.md (hunk 6 above)
Comment (EstEarth202) [reply src/conventions/datapack_advancement.md]:
```suggestion
**Regular format**

```

File: src/conventions/datapack_advancement.md [hunk 11]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
...
+
+**3. Datapack Advancement**
+
+Choose the normal syntax when you have more than one datapack under the namespace.
+
+> the advancement *should* be inside `/data/<namespace>/advancements/<datapack name>.json`.
+
+Example Datapack: [Boomber360](https://www.planetminecraft.com/member/boomber360/).
+
+**Extra: Standalone Datapack**
```
Comment (EstEarth202) [inline src/conventions/datapack_advancement.md]:
```suggestion
**Standalone format**
```

File: src/certified_datapacks/requirement.md (hunk 3 above)
Comment (EstEarth202) [reply src/certified_datapacks/requirement.md:27]:
```suggestion
- **Learn more about the convention** by reading the "Additional Guideline (Optional)" section, this allows you to confidently inspect your datapack before requesting review and reduces the Datapack Reviewer's workload.
- **Request review:** please include every website where your datapack is published. It doesn't have to be just GitHub or Planetminecraft. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
  - **Review type:** please also specify the type of review you require.
    -  **Certified Compatibility ✅ Only**
    -  **Certified Compatibility ✅ + Style Guideline Followed 📚 (Optional)** 
  - **After certified:** then your datapack will have an updated version, which you can request to review as many times as you want.
  - **Request again:** please add the *changelog* or the *location of the changed file*. And If you're using GitHub, we can easily check the commit history. By doing so, the reviewer can't waste time reviewing what they've already reviewed.
  - **Datapack unreleased:** please don't DM the file; instead, use *GitHub private repo* and invite the reviewer. Don't worry about any datapack functions leaking.
- If you have any questions about convention, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```

File: src/style_guideline/reloading_message.md (hunk 10 above)
Comment (EstEarth202) [reply src/style_guideline/reloading_message.md:32]:
```suggestion
### **Exception cases**

**Case 1:** Custom advancement 
- Simply use this a custom advancement to keep track of who has seen the message.

```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```

**Case 2:** Debug message
- This example is debugging reload messages that normal players shouldn't see. 

```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```

Comment (EstEarth202) [reply src/style_guideline/reloading_message.md:32]:
```suggestion
### **Exception cases**

> **Case 1:** a custom advancement to keep track of who has seen the message.
```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```
> **Case 2:** debugging reload messages that normal players shouldn't see.

``convention:datapack/setup``
```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```

File: src/style_guideline/reloading_message.md [hunk 12]
```diff
@@ -2,4 +2,31 @@
...
+  "criteria": {
+    "trigger": {
+      "trigger": "minecraft:tick"
+    }
+  },
+  "rewards": {
+    "function": "..."
+  }
+}
+```
```
Comment (EstEarth202) [inline src/style_guideline/reloading_message.md:25]:
```suggestion
### **Exception cases**

**Case 1:** Custom advancement 
- Simply use this a custom advancement to keep track of who has seen the message.

```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```
```

File: src/style_guideline/reloading_message.md (hunk 10 above)
Comment (EstEarth202) [inline src/style_guideline/reloading_message.md:32]:
```suggestion

**Case 2:** Debug message
- This example is debugging reload messages that normal players shouldn't see. 

```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```

Comment (EstEarth202) [reply src/style_guideline/reloading_message.md]: 📚: fix suggested

File: src/conventions/datapack_advancement.md [hunk 13]
```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
...
+
+Use standalone format when you are going only to have one datapack under the namespace.
+
+> Ex. `/data/global/advancements/standalone/<datapack name>.json`.
```
Comment (EstEarth202) [inline src/conventions/datapack_advancement.md]:
```suggestion
> Ex. `/data/global/advancements/standalone/<project name>.json`.
```

File: src/style_guideline/reloading_message.md (hunk 9 above)
Comment (oOBoomberOo) [reply src/style_guideline/reloading_message.md:9]: No.

File: src/conventions/datapack_advancement.md (hunk 8 above)
Comment (oOBoomberOo) [reply src/conventions/datapack_advancement.md:122]: No.

**Review by oOBoomberOo — CHANGES_REQUESTED** [review]

File: src/style_guideline/reloading_message.md (hunk 10 above)
Comment (oOBoomberOo) [reply src/style_guideline/reloading_message.md]: This still isn't fixed.

File: src/certified_datapacks/requirement.md (hunk 3 above)
Comment (oOBoomberOo) [reply src/certified_datapacks/requirement.md:27]: No improvement from requested changes.

File: src/certified_datapacks/requirement.md (hunk 2 above)
Comment (oOBoomberOo) [reply src/certified_datapacks/requirement.md:10]: Still poorly worded, and not only were you using too much code block, now you are using too many highlights.
